_posts/2015-02-16-microservices.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: Microservices FTW
+author: arto
+excerpt: How to boot up and get some feedback fast
+---
+
+My name is Arto Santala. I've worked in many roles, including software developer, engineer, project manager, scrum master, and architect, as well as a trainer/coach - in companies like Tieturi, ICL, Finnair, and many others. While I've been involved in producing software since the mid-90's, and even before that, I came as new employee in Solita early this year, so it's an exciting new step for me.
```

Comment:
I came in as a new employee to Solita

come ... in - lienee ok merkityksessä "otettiin Solitalle"
tai sitten pelkkä come ... to